btcjson/walletsvrcmds.go

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2014 The btcsuite developers
+// Copyright (c) 2014-2016 The btcsuite developers
+// Copyright (c) 2015-2016 The Decred developers
```

Comment:
Once the command definition and supporting code is moved (see below), there would no longer be any need to update the copyright header in this file since it will have no changes.